packages/logger/logger.pony

```diff
@@ -149,10 +154,10 @@ interface val LogFormatter
 
   See `DefaultLogFormatter` for an example of how to implement a LogFormatter.
   """
-  fun apply(msg: String, loc: SourceLoc): String
+  fun apply(lvl: LogLevel, msg: String, loc: SourceLoc): String
 
 primitive DefaultLogFormatter is LogFormatter
-  fun apply(msg: String, loc: SourceLoc): String =>
+  fun apply(lvl: LogLevel, msg: String, loc: SourceLoc): String =>
```

Comment:
should the default print the log level?

Reply:
I'm leaning toward no. I think it violates expectations slightly by printing the log level used in `create()`, rather than the log level used in the boolean before logging. E.g:

```pony
let logger = StringLogger(Info, env.out)
logger(Error) and logger.log("blah") // would print "INFO: blah"
```

Reply:
I'm leaning towards no as well.